python/test/unit/language/test_line_info.py

```diff
@@ -7,6 +8,7 @@
 import triton
 import triton.language as tl
 from triton._internal_testing import is_interpreter
+from triton._filecheck import run_filecheck
```

Comment:
Test failures are because you've changed the line numbers of everything below this :)